### DISCUSSION TITLE
Open Copilot automatically in sidebar

### DISCUSSION THREAD

**Discussion by LeonelRFF** [logancyang/obsidian-copilot#924] [Q&A]
Hello, I have a question: Is it possible to open Copilot in the sidebar automatically after applying one of my custom prompts?
Let me explain

1. I'm in my note normally
2. I select a paragraph
3. I apply a custom prompt

After the above, the plugin is running in the sidebar, but it has to be opened manually (I still have a hotkey 😉), my idea was to do something like in the edge browser, when selecting a text and right clicking the option "Ask Copilot" appears and the right panel opens automatically.

On the other hand, in the "Open Plugin In" configuration and I select "editor" when using a custom prompt in my note, it automatically opens the tab with the chat 👍 but, if I go back to my note, select another paragraph and run another prompt again, it does not return to the chat tab, I say this last thing about automatically passing because it could be useful when using it on phones

some topics related to this according to what I searched:
- #749
- #166
- #845

**Comment by logancyang** [comment]
@LeonelRFF thanks for the detailed description! 

cc @Emt-lin, this is something you mentioned before
